cranelift/isle/isle/src/sema.rs

```diff
@@ -1782,7 +1782,7 @@ impl TermEnv {
                 let tid = match self.term_map.get(&name) {
                     Some(t) => t,
                     None => {
-                        tyenv.report_error(pos, format!("Unknown term in pattern: '{}'", sym.0));
+                        tyenv.report_error(pos, format!("Unknown term in expression: '{}'", sym.0));
```

Comment:
I suppose if `bindings.vars.iter().any(|b| b.name == sym)`, then you could extend the error message to note that there is a variable by that name. That might have helped in the case that brought this mistake to your attention.

Reply:
Great idea, thanks! Just added.